vscode/src/webviewPanel.ts

```diff
@@ -529,7 +529,13 @@ export class QSharpWebViewPanel {
   }
 
   reveal(column: ViewColumn) {
-    this.panel.reveal(column, true);
+    // If it's already visible, don't do anything else it messes up the existing layout
+    // This isn't perfect, as if the tab is just in the background it will still call
+    // reveal to bring to the foreground and change the layout, but this is the best we
+    // can do with the current API.
```

Comment:
Played around with it a bit, and couldn't get it to mess up the layout. Is there a repro for it still changing the layout in a way that a user wouldn't expect?